README.md

```diff
@@ -28,6 +28,16 @@ The interactive CLI can guide you through the following steps:
 npx esperf
 ```
 
+## Advanced Usage
+
+The interactive CLI will show more steps:
```

Comment:
like mentioned in the other thread, i'd just have the thread count as a flag

either `parallelism` or `num-threads` maybe. so it'd never be an interactive choice, and always a flag

Reply:
Yeah just saw, perfect, will do that.